app/src/main/java/org/openedx/app/AppViewModel.kt

```diff
@@ -36,7 +38,8 @@ class AppViewModel(
     private val analytics: AppAnalytics,
     private val deepLinkRouter: DeepLinkRouter,
     private val fileUtil: FileUtil,
-    private val context: Context
+    private val context: Context,
```

Comment:
Push Manager would be a better choice here. We should have marked things internal for notification module.

Reply:
addressed.